src/dmclock_server.h

```diff
@@ -1752,6 +1752,7 @@ namespace crimson {
 	default:
 	  assert(false);
 	}
+        return next_req.type;
```

Comment:
nit, might want to use tab for indent to be consistent with the rest of this file

Reply:
I've adjusted the indentation.